docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10.5
+    container_name: djangoplicity-customsearch-db
+    restart: always
+    environment:
+      POSTGRES_DB: djangoplicity-customsearch
+      POSTGRES_USER: djangoplicity-customsearch
+      POSTGRES_PASSWORD: djangoplicity-customsearch
+    ports:
+      - "5433:5432"
+
+  web:
+    build: .
+    container_name: djangoplicity-customsearch-web
+    command: ["./scripts/command-dev.sh"]
+    volumes:
+      - ./djangoplicity:/app/djangoplicity
+      - ./testapp/:/app/testapp
+      - ./htmlcov:/app/htmlcov
+      - ./.coveragerc:/app/.coveragerc
+      - ./db:/app/db
```

Comment:
This db volume was intended to save a sqlite database, but as you are using postgres then it's not required, you should rather create a named volume for the postgres database.